puppet/zulip_ops/manifests/base.pp

```diff
@@ -19,9 +19,8 @@
                         # Needed for zulip-ec2-configure-network-interfaces
                         "python3-six",
                         "python-six",
-                          # This one is needed for postgres as well
                         # "python3-boto", # missing on trusty
```

Comment:
Here is a possible solution to sidestep the puppet system packages availability issue: https://travis-ci.org/rht/zulip/builds/263380620, which then can be turned to 100% Python 3.